### DISCUSSION TITLE
Any plans to support .heic files (format HEIF)?

### DISCUSSION THREAD

**Discussion by igormcoelho** [linuxdeepin/developer-center#2913] [Features Request & Ideas | 特性请求 & 头脑风暴]
It would be very nice to have deepin image viewer supporting .heic files, since iOS have them as default nowadays and it's awkward to manually convert them fto .jpg for a simple visualization.
Currently, only GIMP software is able to open this file, but it's not good for regular users and batches of pictures.

**Comment by dependon** [comment]
At present, we are still making it popular, and we have no support at present, but thank you for your suggestion

**Comment by BLumia** [comment]
3rd party Qt image formats plugin could be used:

- https://github.com/jakar/qt-heif-image-plugin
- https://github.com/novomesk/qt-heic-image-plugin

There is also a related bug report:

- https://bugreports.qt.io/browse/QTBUG-69212

btw I'm not 100% sure but we **might** have (HEVC encoding-related) patent issues to introduce such support.
